Tests/SonarQube.TeamBuild.PreProcessor.Tests/WebClientDownloaderTest.cs

```diff
@@ -41,55 +41,22 @@ public void Credentials()
         [TestMethod]
         public void SemicolonInUsername()
         {
-            try
```

Comment:
The tests perform the same checks, but use "AssertException.Expects" instead of explicitly using a try-catch